### DISCUSSION TITLE
Background Color Error

### DISCUSSION THREAD

**Discussion by LeoEvaldt** [requarks/wiki#6649] [Error / Bug Report]
Hi everyone.

When I try to change the background color for a table, only some of the cells change the color.

Thanks for any help.

wiki.js version: 2.5.299

![Bug backgroud color wiki js](https://github.com/requarks/wiki/assets/141968714/29e49c5a-1c5a-4ffc-9161-788b9990368d)

**Comment by j842** [comment]
Hitting the same issue... Using `!important` works in the preview, but not in the main rendered version... e.g.
```
<style>
  td:nth-child(2) {background: white !important;}
  td:nth-child(3) {background: white !important;}
  td:nth-child(4) {background: black !important;}
  td:nth-child(5) {background: black !important;}
  </style>
```
